examples/client/lwm2mclient.c

```diff
@@ -478,6 +543,13 @@ static void prv_object_list(lwm2m_context_t * lwm2mH,
     }
 }
 
+/**
+ * @brief Dumps the TLV data for a given instance of an object.
```

Comment:
What is TLV data? This should be described, both with a comment on what does `dump_tlv` do.